### PR TITLE
DL-6802-ATED Update ATED statement inline with latest DAC

### PR DESCRIPTION
**Pull request by russellthorn**

### CODE REVIEW

File: conf/services/ated.yml [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
 serviceDomain: www.tax.service.gov.uk
 serviceUrl: /ated/home
 contactFrontendServiceId: ATED
-complianceStatus: noncompliant
+complianceStatus: partial
+accessibilityProblems:
+  - the required format for the input is unclear.
+  - fieldset is missing a legend.
+  - links were not descriptive.
+  - focus order becomes illogical when the sercice timeout is appears.
```
Comment (paulanderson1234) [inline conf/services/ated.yml:14]: Spelling mistake of 'service' ?

Comment (russellthorn) [reply conf/services/ated.yml:14]: good spot, fixed now 👍

Comment (russellthorn) [reply conf/services/ated.yml:14]: I have just pushed a commit to correct this, but it's not showing

Comment (paulanderson1234) [reply conf/services/ated.yml:14]: Still needs attention as it does not make sense : focus order becomes illogical when the service timeout is appears.

**Review by paulanderson1234 — APPROVED** [review]

File: conf/services/ated.yml (hunk 1 above)
Comment (paulanderson1234) [reply conf/services/ated.yml]: Reviewed and checked locally - updated

**Pull request merged**